src/tools/rbd/action/Bench.cc

```diff
@@ -16,13 +16,19 @@
 
 namespace rbd {
 namespace action {
-namespace bench_write {
+namespace bench {
 
 namespace at = argument_types;
 namespace po = boost::program_options;
 
 namespace {
 
+enum io_type {
+  bench_read = 0,
```

Comment:
Minor: enums should be named like constants (e.g. IO_TYPE_READ and IO_TYPE_WRITE)